worksheet_b/README.MD

```diff
@@ -0,0 +1,53 @@
+
+
+Print Welcome to
+Terminal 4
+
+Print Password
+Required
+
+Secret Word = Water
+
+Player Strikes Left
+= 5
+
+Computer generates
+the 11 words in a random order
+
+                Computer words = ("Bales,
+Towel, Water, Artsy, Fairy, Chair, Wacky, Voice, Dough,            Grass, Knee, Water.")
+
+If Player word is
+not equal to secret word player loses strike 
+
+                if Letters = same position as
+Secret Word display similiarity 
```

Comment:
The worksheet asked you to give pseudocode for computing the similarity, i.e. loop over the words and count the number of matching letters.